routers/routers.go

```diff
@@ -1,38 +1,38 @@
-package routers
-
-import (
-	"github.com/atmiguel/cerealnotes/handlers"
-	"github.com/atmiguel/cerealnotes/paths"
-	"net/http"
-)
-
-func SetRoutes() {
-	// static files
-	{
-		staticDirectoryName := "static"
-		staticDirectoryPaddedWithSlashes := "/" + staticDirectoryName + "/"
-
-		fileServer := http.FileServer(http.Dir(staticDirectoryName))
-
-		http.Handle(
-			staticDirectoryPaddedWithSlashes,
-			http.StripPrefix(staticDirectoryPaddedWithSlashes, fileServer))
-	}
-
-	// templates
-	http.HandleFunc(paths.LoginOrSignupPath, handlers.HandleLoginOrSignupRequest)
-
-	// forms
-	http.HandleFunc("/user", handlers.HandleUserRequest)
-	http.HandleFunc("/session", handlers.HandleSessionRequest)
-
-	// requires Authentication
-	handleAuthenticated(paths.HomePath, handlers.HandleHomeRequest)
-}
-
-func handleAuthenticated(
-	pattern string,
-	handlerFunc handlers.AuthentictedRequestHandlerType,
-) {
-	http.HandleFunc(pattern, handlers.AuthenticateOrRedirectToLogin(handlerFunc))
-}
+package routers 
+ 
+import ( 
+	"github.com/atmiguel/cerealnotes/handlers" 
+	"github.com/atmiguel/cerealnotes/paths" 
+	"net/http" 
+) 
+ 
+func SetRoutes() { 
```

Comment:
I'm not a big fan of `SetRoutes()` setting global state. In main.go you call `http.ListenAndServe(port, nil)`, and it's not easy for me to figure out where the handler is coming from. I'd prefer something like
```go
// In routers.go
func GetRoutes() http.Handler {
  mux := http.NewServeMux()
  mux.HandleFunc(paths.LoginOrSignupPath, handlers.HandleLoginOrSignupRequest)
  mux.HandleFunc(paths.LoginOrSignupPath, handlers.HandleLoginOrSignupRequest)
  mux.HandleFunc("/user", handlers.HandleUserRequest)
  mux.HandleFunc("/session", handlers.HandleSessionRequest)
  return mux
}

// In main.go
func main() {
  handler := routers.GetRoutes()
  http.ListenAndServe(port, handler)
}
```

GetRoutes might be a terrible name, but you get the idea